docs/content/commands/npm-uninstall.md

```diff
@@ -7,7 +7,7 @@ description: Remove a package
 ### Synopsis
 
 ```bash
-npm uninstall [<@scope>/]<pkg>[@<version>]... [-S|--save|-D|--save-dev|-O|--save-optional|--no-save]
+npm uninstall [<@scope>/]<pkg>[@<version>]... [--save|--no-save]
```

Comment:
```suggestion
npm uninstall [<@scope>/]<pkg>[@<version>]... [-S|--save|--no-save]
```
not sure if this matters or not tho

Reply:
I checked arborist and it did not appear to be parsing the `-S` option anywhere, just explicit `--save`

Reply:
Yep, that shorthand option still works too